ros_ws/src/depthai_ros/include/depthai_ros/depth_map_publisher_node.h

```diff
@@ -52,7 +56,9 @@ class DepthMapPublisherNode
     std::string landmark_topic_;
     std::string disparity_service_name_;
     ros::Publisher depth_map_pub_;
+    ros::Publisher landmarks_pub_;
     std::unique_ptr<DepthAI::DepthAI> oak_;
```

Comment:
How come you do not include the `depthai_wrapper.hpp` file from the `depthai` here? How does this file know where `DepthAI` object is from?